inference/inference.py

```diff
@@ -311,17 +323,26 @@ def autoregressive_inference(params, ic, valid_data_full, model):
             f'global_mean_gradient_magnitude_target/ic{ic}/channel{c}-{name}':
             gradient_magnitude_target[i, c] for c, name in enumerate(out_names)
           }
-          wandb.log(
-            {
-              **rmse_metrics,
-              **acc_metrics,
-              **mean_pred_metrics,
-              **mean_target_metrics,
-              **grad_mag_pred_metrics,
-              **grad_mag_target_metrics
-            }
-          )
+          if params.is_log_time_series_metrics_to_wandb:
+            wandb.log(
+              {
+                **rmse_metrics,
+                **acc_metrics,
+                **mean_pred_metrics,
+                **mean_target_metrics,
+                **grad_mag_pred_metrics,
+                **grad_mag_target_metrics
+              }
+            )
               
+    if params.is_log_aggregated_metrics_to_wandb:
+      # each metric is [time x channel]
+      all_metrics = [valid_loss, acc, global_mean_pred, global_mean_target, gradient_magnitude_pred, gradient_magnitude_target]
+      time_avg_metrics = [m.mean(axis=0) for m in all_metrics]
```

Comment:
Let's do snapshots at 5 days and 10 days, instead of time-mean.

Reply:
Should be done. Thanks!